docs/using-keycloak.md

```diff
@@ -32,12 +32,14 @@ docker run -d --restart=always \
 
 Then run the actual Keycloak server, using the MySQL-compatible
 version of
-[the Keycloak images](https://github.com/jboss-dockerfiles/keycloak)
+[the Keycloak images](https://hub.docker.com/r/jboss/keycloak/)
```

Comment:
After this change, the link no longer refers to a list containing one MySQL-compatible version. Could you rephrase the sentence to reflect this, e.g. just calling it *the* image?

Reply:
Fixed :)